.stickler.yml

```diff
@@ -0,0 +1,16 @@
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    # indicate where is the config file for stylelint
+    config: './eslint.config'
+
+# add the files here you want to be ignored by stylelint
+files:
+  ignore:
+      - 'dist/*'
+      - 'src/*'
```

Comment:
You can't ignore all the file in `src`, what is the access of linter's check ? 🤔 
kindly remove this.